pom.xml

```diff
@@ -62,7 +62,7 @@
         <dep.nexus-staging-plugin.version>1.6.8</dep.nexus-staging-plugin.version>
         <dep.asm.version>6.2.1</dep.asm.version>
         <dep.gcs.version>1.9.17</dep.gcs.version>
-        <dep.alluxio.version>2.1.2</dep.alluxio.version>
```

Comment:
This seems unrelated with this PR? Maybe we could put it into a separate pull request.